apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx

```diff
@@ -100,6 +106,100 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
         description: "Subtitle for applicant dashboard hero",
       })}
     >
+      {searchParams.get(FromIapDraftQueryKey) === "true" && (
+        <Alert.Root
+          type="info"
+          dismissible
+          onDismiss={() => {
+            searchParams.delete(FromIapDraftQueryKey);
+            setSearchParams(searchParams);
+          }}
```

Comment:
This is subjective, but I'm leaning towards these not being live alerts. I don't think it would be helpful to interrupt the user with this information 🤔 

```suggestion
        <Alert.Root
          type="info"
          dismissible
          live={false}
          onDismiss={() => {
            searchParams.delete(FromIapDraftQueryKey);
            setSearchParams(searchParams);
          }}
```

Reply:
I don't have enough experience with AT to have an opinion on this so I'll take your suggestion.  :sweat_smile:  As far as I know, the main motivation for these notifications is the theme change which would not be as jarring to folks relying on narration, I imagine.
Ref: [remove live alert](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6750/commits/fd8567e2731c70204bf842b6d6e21cd792e8cb63)